Docs/Accelerator.md

```diff
@@ -42,6 +42,18 @@ BEGIN
  M, 419, VIRTKEY , NOINVERT , ALT
 END
 ```
+**How to interpret these values, with the following example:**
+
+`VK_TAB, 41008, VIRTKEY , NOINVERT , CONTROL`
+
+The Accelerator will excecute command 41008, when the User presses Ctrl + Tab
+
+| Entry        | Explatation           | Type  |
+| ------------- |:-------------:| -----:|
+| `VK_TAB`      | key top press for Accelerator | Virtual Key or ASCII character |
+| `41008`      | command / identifier to execute      |   Integer-Value |
+| `VIRTKEY , NOINVERT , CONTROL` |   options    |    string, entries separated with ',' |
```

Comment:
No need to align column 3 to the right.

Cleanup the Capitals vs. lowercase things. In the last column for example you say `string`, which should probably be `String`. 

`Integer-Value` doesn't sound like a type to me, maybe just `Integer`? 

I think it's either `Virtual Key or ASCII Character` or `virtual key or ASCII character`